interface/patient_file/encounter/delete_form.php

```diff
@@ -82,7 +85,17 @@
 ?>
 <input type="hidden" id="confirm" name="confirm" value="1"/>
 <p>
-<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($_GET['formname'])); ?>
+
+    <?php
+    $result = getFormByEncounter(
+        $pid,
+        $_REQUEST['encounter'],
+        "id, date, form_id, form_name, formdir, user, deleted",
+        "",
+        "FIND_IN_SET(formdir,'newpatient') DESC, form_name, date DESC",
+        $_REQUEST['id']
+    );
+    echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($result[0]['form_name'])); ?>
```

Comment:
Couldn't we instead pass the form_name parameter via the delete_form.php call in line 1049 of interface/patient_file/encounter/forms.php

Reply:
Also, as an aside, if are collecting a form and have an id, then could just use the getFormById() function instead.

Reply:
I used : 
getFormNameByFormdir($_REQUEST["formname"])["form_name"]);